src/entity/Comment.ts

```diff
@@ -50,7 +50,7 @@ export class Comment {
   featured: boolean;
 
   @Column({ type: 'jsonb', default: {} })
-  @Index('IDX_user_flags_vordr', { synchronize: false })
```

Comment:
Copy-pasta issue. Confirmed from the migration file on what should be the right name. Should not impact anything since we have `synchronize: false`.